deeplearning4j-ui-parent/deeplearning4j-play/src/main/java/org/deeplearning4j/ui/play/PlayUIServer.java

```diff
@@ -79,13 +81,19 @@
     private final AtomicBoolean shutdown = new AtomicBoolean(false);
 
     private Thread uiEventRoutingThread;
-    @Parameter(names = {"-r", "-enableRemote"}, description = "Whether to enable remote or not", arity = 1)
-    private boolean enableRemote;
 
+    @Parameter(names = {"-r", "--enableRemote"}, description = "Whether to enable remote or not", arity = 1)
+    private boolean enableRemote;
 
-    @Parameter(names = {"--uiPort"}, description = "Whether to enable remote or not", arity = 1)
+    @Parameter(names = {"-p", "--uiPort"}, description = "Whether to enable remote or not", arity = 1)
     private int port = DEFAULT_UI_PORT;
 
+    @Parameter(names = {"-fs", "--enableFileStorage"}, description = "Use file-based stats storage", arity = 1)
+    private boolean enableFileStorage;
+
+    @Parameter(names = {"--customStatsFile"}, description = "Path to create custom stats file", arity = 1)
```

Comment:
arity=1 I think is only valid for boolean args. Also improve description - " for remote listener support. If set, also enables remote listener."

Reply:
incorrect, the javadoc says

> How many parameter values this parameter will consume. For example, an arity of 2 will allow "-pair value1 value2".